model/span.go

```diff
@@ -101,7 +101,7 @@ func (s *Span) NormalizeTimestamps() {
 func (s *Span) ParentSpanID() SpanID {
 	for i := range s.References {
 		ref := &s.References[i]
-		if ref.TraceID == s.TraceID && ref.RefType == ChildOf {
+		if ref.TraceID == s.TraceID && (ref.RefType == ChildOf || ref.RefType == FollowsFrom) {
```

Comment:
if no unit tests break I think it's reasonable. But we always treated ChildOf as higher priority, so a better fix would be (a) search for ChildOf ref, (b) if not found, search for FollowsFrom() (this can be optimized to be done in one loop)

Reply:
that makes sense, I will make adjustments